src/main/java/seedu/address/model/datetime/DatetimeCommonUtils.java

```diff
@@ -20,44 +24,51 @@ public class DatetimeCommonUtils {
     public static final String DATE_FORMAT_REGEX = "\\d{4}-\\d{2}-\\d{2}";
     public static final String DATE_READABLE_FORMAT = "yyyy MMM dd";
```

Comment:
It should be `MM` instead of `MMM` ?
btw, I think `mm` is more consistent with `yyyy` and `dd`

Reply:
I'm using the Java format:
M: 7
MM: 07
MMM: Jul